target/linux/ramips/image/mt7621.mk

```diff
@@ -2173,6 +2173,16 @@ define Device/zio_freezio
 endef
 TARGET_DEVICES += zio_freezio
 
+define Device/sztozed_zlt-s12-pro
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  IMAGE_SIZE := 16064k
+  DEVICE_VENDOR := ZLT
+  DEVICE_MODEL := S12 PRO
+  DEVICE_PACKAGES := kmod-mt7603 kmod-mt76x2 kmod-usb3 kmod-usb-serial-option comgt-ncm
```

Comment:
kmod-usb-serial-option is redundant here, as it is pulled along with the comgt-ncm package